lib/hapi.js

```diff
@@ -1,37 +1,37 @@
-// var _ = require('lodash')
-// var PouchDB = require('pouchdb')
+var assign = require('lodash').assign
 
-module.exports = function (config, usersDb, callback) {
-  // var users = new PouchDB(usersDb)
-  // users.plugin(require('pouchdb-users'))
-  // users.installUsersBehavior()
-  // .then(function () {
-  //   var defaultOpts = {
-  //     config: config,
-  //     database: require('./database')(config),
-  //     prefix: '/hoodie/account',
-  //     users: users
-  //   }
-  //
-  //   // insert below code here
-  // }, callback)
+module.exports = function (config, usersDbName, callback) {
+  var database = require('./database')(config)
+  var usersDb = database(usersDbName)
 
-  callback(null, [
-    require('inert'),
-    require('h2o2')
-  ].concat([
-    require('./static'),
-    require('./http-log')
-  ].map(function (plugin) {
-    return {
-      register: plugin,
-      options: {config: config}
-    }
-  })/*, [{
-    register: require('@hoodie/server-account-node-sessions'),
-    options: defaultOpts
-  }, {
-    register: require('hoodie-server-store'),
-    options: _.defaults({prefix: '/hoodie/store'}, defaultOpts)
-  }]*/))
+  usersDb.constructor.plugin(require('pouchdb-users'))
+
+  usersDb.installUsersBehavior().then(function () {
+    var options = {config, database, usersDb}
+
+    var hapiPlugins = [
+      require('inert'),
+      require('h2o2')
+    ]
+
+    var localPlugins = [
+      require('./static'),
+      require('./http-log')
+    ].map(function (register) { return {register, options} })
+
+    var hoodieCorePlugins = [{
+      register: require('hoodie-server-account'),
+      prefix: '/hoodie/account'
```

Comment:
I meant the name attribute: https://github.com/hoodiehq/hoodie-server-store/blob/master/index.js#L3